scripts/curator.py

```diff
@@ -105,57 +95,43 @@ def get_actionable_indices(es, max_allowed_size, index_name_prefixes_list):
     Returns a list of actionable indices based on percentage size and age.
     """
     size_counter = 0
+    # Prepare dictionary of indices with their size and creation_date values.
+    data = {}
```

Comment:
Is there a specific reason to use a map instead of an array for this? I think handling below could be easier if it was an array of objects of the form
```
{
   "name": <index-name>
   "size": <size>
   "creation_date": <date>
}
```